apps/discovery-platform/src/db/index.ts

```diff
@@ -168,17 +168,44 @@ export const createQuota = async (
   dbInstance: DBInstance,
   quota: Quota
 ): Promise<QuotaDB> => {
-  const text = `INSERT INTO ${TABLES.QUOTAS} (id, client_id, paid_by, quota, action_taker)
-  VALUES (default, $<clientId>, $<paidBy>, $<quota>, $<actionTaker>) RETURNING *`;
-  const values: Quota = {
+  const quotaPaidIncrement = quota.quota >= BigInt(0) ? quota.quota : BigInt(0);
```

Comment:
should we enforce a quota value outside of this? Because running this transaction with zero does not make sense?
Or am I missing a scenario where zero would be a wanted increment?